pkg/webhook/github.go

```diff
@@ -94,6 +95,20 @@ func (s *githubHook) handleEvent(c *gin.Context, eventType string) {
 		}
 		repo = e.Repo.GetFullName()
 		commit = e.PullRequest.Head.GetSHA()
+	case *github.CommitCommentEvent:
+		repo = e.Repo.GetFullName()
+		commit = *e.Comment.CommitID
```

Comment:
We may need to validate what CommitID actually is.  For example you can comment on a single commit in a pull request.

Reply:
Yeah, I'm not actually sure how we want to handle this one. In most cases, it sorta seems like a useless hook to have. I almost removed it... then I figured I might as well leave it, since somebody somewhere is gonna have a use for it.